master/reference/calicoctl/commands/get.md

```diff
@@ -48,6 +48,9 @@ Options:
                                Uses the default namespace if not specified.
   -a --all-namespaces          If present, list the requested object(s) across 
                                all namespaces.
+  --export                     If present, returns the requested object(s) stripped of
+                               cluster-specific information. This flag will be ignored
+                               if <NAME> is not specified.
```

Comment:
I think this should be `if resource name is not specified.` Maybe @emanic has a better suggestion

Reply:
Well, `<NAME>` matches what is up on the top. However, nowhere is it explained what value they should use for `<NAME>` (that I could find). So, perhaps it could be added somewhere what kind of value to use for this. It seems that it should be the name of a resource. Alternatively, since `<NAME>` is pretty vague, you could change it to `<RESOURCE-NAME>` and that would probably help a lot.

It's nice that it returns an object, presumably this is the specified resource? Might be nice to tell them what format it gets returned in? I am also a little unclear as to just what cluster specific information gets stripped. I guess that it might be too much information to add into this short space.

It might be helpful to tell them what the flag is intended to be used for, what is the use case? As I understand it, it is designed so that people can get the object, modify it, and then pass it back in via `replace`. If I am correct, then let me know and perhaps I can suggest some alternate wording.

Reply:
I believe the first sentence of the help string is from k8s `--export` message, so I'm guessing people are familiar with that. And, we usually copy the same exact help message we have in the code, that's why we have the `<NAME>` here. I'm not too fussed either way

Reply:
I can remove the second part. That part isn't documented by k8s, but I thought it would be nice to have since it certainly caused me confusion. WDYT?

Reply:
let's mimic k8s, since we also have the same behavior (also will need to change the calicoctl PR message)

Reply:
I think it's good to have the second part, @mgleung